aptos-move/aptos-aggregator/src/delayed_field_extension.rs

```diff
@@ -21,6 +21,7 @@ fn get_delayed_field_value_from_storage(
 ) -> Result<DelayedFieldValue, PanicOr<DelayedFieldsSpeculativeError>> {
     resolver
         .get_delayed_field_value(id)
+        // TODO[agg_v2]: Is this error mapping correct?
```

Comment:
make // TODO[agg_v2](fix) so we know later when we search to check it before everything is ready (i.e. we use either (fix) (clean) (test))